static/templates/partials/iframely-widget-card.tpl

```diff
@@ -10,7 +10,7 @@
 
             <!-- IF title -->
             <h4 class="media-heading">
-                <a href="{embed.url}" target="_blank" rel="nofollow" class="one-line">
+                <a href="{embed.meta.canonical}" target="_blank" rel="nofollow" class="one-line">
```

Comment:
Will need confirmation from @nleush re: this change, but I recall you talking about it on your forum :y:

In what circumstances is `url` and `meta.canonical` different or unset?